submissions/akulahere/html-css-popup/css/main.css

```diff
@@ -0,0 +1,138 @@
+body {
+    font-family: "Times New Roman";
+    margin: 0px;
+}
+
+a {
+    text-decoration: none;
+    color:black;
+}
+
+img {
+    height: 30px;
```

Comment:
Use max-height or set height in %, because icons in the pop-up are stretched

Reply:
fixed